life_expectancy/tests/test_country_enum.py

```diff
@@ -0,0 +1,9 @@
+"""
+module that tests the class function created in the country enum
+"""
+from life_expectancy.country import Country
+
+def test_country_list(output_expected_country_list):
```

Comment:
is this really a test on want to make? From my understanding you would rather want to check if the inputed country by the user corresponds to one of the possible country options, which would be done on main.py

Reply:
From what I understood of the last "alinea" was to test the created function country-to-list